frontend/src/wizards/Governance/Policy/PolicyWizard.tsx

```diff
@@ -853,7 +853,7 @@ function ObjectTemplate() {
           placeholder={t('Add rule')}
           collapsedPlaceholder={t('Expand to edit')}
         >
-          <WizStringsInput label={t('API Groups')} path="apiGroups" />
+          <WizStringsInput required={false} label={t('API Groups')} path="apiGroups" />
           <WizStringsInput label={t('Resources')} path="resources" />
           <WizStringsInput label={t('Verbs')} path="verbs" />
```

Comment:
To maintain existing behaviour for every other `WizStringsInput` instance, I think you'll need to add the `required` prop? I think there should be no need to add it on line 856 now, as it will not be required by default.

Reply:
Ack. Brain glitch.